sql/plan.go

```diff
@@ -50,6 +49,10 @@ func (p *planner) resetTxn() {
 // plan needs to be iterated over using planNode.Next() and planNode.Values()
 // in order to retrieve matching rows.
 func (p *planner) makePlan(stmt parser.Statement) (planNode, error) {
+	if stmt.StatementType() == parser.DDL {
+		p.txn.SetSystemDBTrigger()
```

Comment:
This will set the system DB trigger for DDL statements that have no effect, such as `CREATE TABLE IF NOT EXISTS foo (...)` where table `foo` already exists. Is that a concern? If it's not, add a comment here explaining why.